packages/gatsby-source-wordpress/src/fetch.js

```diff
@@ -355,21 +356,40 @@ async function getPages(
   }
 }
 
+/**
+ * Check a route against the whitelist or blacklist
+ * to determine validity.
+ *
+ * @param {any} routePath
+ * @param {Array} routeList
+ * @returns {boolean}
+ */
+function checkRouteList( routePath, routeList ) {
+  if (
+    routeList.some(route =>
+      minimatch(routePath, route)
+    )
+  ) {
+    return true
+  } else {
+    return false
+  }
+}
```

Comment:
`Array.prototype.some` returns `true` or `false` so this could be shorter:
```js
function checkRouteList( routePath, routeList ) {
  return routeList.some(route =>
    minimatch(routePath, route)
  )
}
```

Reply:
Good catch :smiley: